src/main/java/org/gridsuite/study/server/repository/StudyEntity.java

```diff
@@ -89,13 +89,8 @@ public class StudyEntity extends AbstractManuallyAssignedIdentifierEntity<UUID>
     @Column(name = "voltageInitParametersUuid")
     private UUID voltageInitParametersUuid;
 
-    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
-    @JoinColumn(name = "securityAnalysisParametersEntity_id",
-            referencedColumnName = "id",
-            foreignKey = @ForeignKey(
-                    name = "securityAnalysisParameters_id_fk"
-            ))
-    private SecurityAnalysisParametersEntity securityAnalysisParameters;
+    @Column(name = "securityAnalysisParametersUuid")
+    private UUID securityAnalysisParametersUuid;
```

Comment:
We must keep the old table during the migration of parameters to the service
```suggestion
    @Deprecated(forRemoval = true)
    @Getter(AccessLevel.PROTECTED)
    @Setter(AccessLevel.PROTECTED)
    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
    @JoinColumn(name = "securityAnalysisParametersEntity_id",
            referencedColumnName = "id",
            foreignKey = @ForeignKey(
                    name = "securityAnalysisParameters_id_fk"
            ))
    private SecurityAnalysisParametersEntity securityAnalysisParameters;

    @Column(name = "securityAnalysisParametersUuid")
    private UUID securityAnalysisParametersUuid;
```